_team/20-brigit-anderson.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: person
+category: team
+permalink: /team/brigit-anderson/
+
+name: Brigit Anderson
+position: GIS/IM Officer
+image: img\team\brigit-anderson.jpg
```

Comment:
windows file paths have the `\` and while i think a web browser will flip these around, i think this should match URL format and use `/`